pkg/plugins/policies/policies.go

```diff
@@ -22,3 +22,7 @@ var policies = []plugins.PluginName{
 func Enabled() []plugins.PluginName {
 	return policies
 }
+
+func SetPolicies(newPolicies []plugins.PluginName) {
```

Comment:
We can make extend this by code or by configuration. I think we agreed that we don't want to expose the configuration but maybe it's worth it if there is another plugin in the repository extending the current.

Reply:
> I think we agreed that we don't want to expose the configuration but maybe it's worth it if there is another plugin in the repository extending the current.

Do you mean it's worth doing more than this PR does?

Reply:
why not just make te var public and ditch the `Enabled()` and `SetPolicies()`

Reply:
> > I think we agreed that we don't want to expose the configuration but maybe it's worth it if there is another plugin in the repository extending the current.
> 
> Do you mean it's worth doing more than this PR does?

I thought that maybe doing configuration would be easier for other projects but even when the code change is required it doesn't seem to be a huge change.